Floofbot/Modules/TagCommands.cs

```diff
@@ -58,10 +121,9 @@ public TagCommands(FloofDataContext floofDb)
                     _floofDb.SaveChanges();
                     await SendEmbed(CreateDescriptionEmbed($"💾 Added Tag `{tag}`"));
                 }
-                catch (DbUpdateException e)
+                catch (DbUpdateException)
```

Comment:
I kind of dont like this, other exception errors could occur regarding the database that are not just entries already existing

Perhaps checking if it exists first is better?